library/sensu_process_check

```diff
@@ -1,35 +1,79 @@
-#!/usr/bin/env ruby
-require 'rubygems'
-require 'antsy'
-
-args = Antsy.args
-service = args[:service]
-short_service_name = File.basename(service)
-warn_over=args[:warn_over] || 15
-crit_over=args[:crit_over] || 30
-
-Antsy.fail! "argument 'service' is required" unless service and not service.empty?
-
-check_fname = "/etc/sensu/conf.d/checks/#{short_service_name}-service.json"
-
-check = {
-  'checks' => {
-    short_service_name => {
-      'command' => "/etc/sensu/plugins/check-procs.rb -p #{service} -w #{warn_over} -c #{crit_over} -W 1 -C 1",
-      'standalone' => true,
-      'handlers' => [ 'pagerduty' ],
-      'interval' => 30,
-      'notification' => "unexpected number of #{service} processes",
-      'occurrences' => 2
-    }
-  }
-}
-
-if File.exists?(check_fname) and (JSON.parse(File.read(check_fname)) == check)
-  Antsy.no_change!
-else
-  File.open(check_fname, 'w') do |f|
-    f.write JSON.pretty_generate(check)
-  end
-  Antsy.changed!
-end
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright 2014, Blue Box Group, Inc.
+# Copyright 2014, Craig Tracey <craigtracey@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import traceback
+
+from hashlib import md5
+from jinja2 import Environment
+
+def main():
+
+    module = AnsibleModule(
+        argument_spec=dict(
+            service=dict(default=None, required=True),
+            short_service_name=dict(default='', required=False),
```

Comment:
The default default is None, and required is default to False, so there shouldn't be a need to add any arguments here to the dict() call.